index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mobile version</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <a class="logo">My logo</a>
+        <button type="button" class="btn">
+          <img alt="nav" class="menu" src="./images/Union.png" />
+        </button>
+      </nav>
+    </header>
```

Comment:
- [ OPTIONAL ] How about we have a skeleton something like this? In my opinion, it will help you easily style the layout of the header section in different screen size later in the module.
```
<header>
  <a > Logo </a>
  <nav>
     <button></button>
  </nav>
</header>
```